src/app/app.component.ts

```diff
@@ -70,12 +71,19 @@ function fixUpNav(nav: any, baseUrl: string) {
   templateUrl: './app.component.html'
 })
 export class AppComponent implements OnInit {
+  public isReady = false;
+
   constructor(
     private router: Router,
     private windowRef: SkyAppWindowRef,
     private config: SkyAppConfig,
+    private styleLoader: SkyAppStyleLoader,
     @Optional() private searchProvider?: SkyAppSearchResultsProvider
-  ) { }
+  ) {
+    this.styleLoader.loadStyles().then(() => {
```

Comment:
I'd prefer to gracefully fail and not show an error message.  Writing something to `console.log` should be sufficient.